samples/bluetooth/direct_test_mode/src/fem/nrf21540.h

```diff
@@ -29,157 +29,6 @@ extern "C" {
  */
 #define NRF21540_PD_PG_TIME_US 18
 
-/**@brief Value used as event zero-address - for immediate function execution.
```

Comment:
I think this header file can be removed and the remaining definitions can be moved to the implementation file

Reply:
You are right. I can get those values from the DTS configuration

Reply:
Done